Video-Transformers-Kotlin/app/src/main/java/com/tokbox/sample/basicvideochat/MainActivity.kt

```diff
@@ -0,0 +1,325 @@
+package com.tokbox.sample.basicvideochat
```

Comment:
This is a sample for video transformers.  Is the convention to always use this package id?  Just curious.

Reply:
Since we start new samples from Basic Video Chat sample, there were some references still. All removed now